modules/lead/toolchain/charts/harbor-volumes/values.yaml

```diff
@@ -1,9 +1,12 @@
+storageClassName:
 components:
   registry:
+    size:
```

Comment:
Should this have a size declared?

Reply:
The size is declared when we consume the helm chart in terraform. This is only to help my IDE with auto complete

Reply:
Awesome, thanks!